kinoml/datasets/kinomescan/utils.py

```diff
@@ -0,0 +1,87 @@
+import re
+import logging
+from pathlib import Path
+import pandas as pd
+
+from ...core.protein import AminoAcidSequence
+from ...utils import grouper, datapath, APPDIR
+
+logger = logging.getLogger(__name__)
+
+
+class KINOMEScanMapper:
```

Comment:
Let's add docstrings for all methods!

Reply:
Tuning the documentation system to start doing this! Can't wait!

Reply:
It's so pretty I don't want to write Python, just markdown: https://openkinome.github.io/kinoml/